### ISSUE TITLE
Regression: cannot create local lists

### ISSUE THREAD

**Issue by kranix0** [michaldrabik/showly#760]
Showly used to support the creation of local lists which were not synced to Trakt.

Now when I try to create a new list, I get an error.

![Screenshot_20231219-170333](https://github.com/michaldrabik/showly-2.0/assets/461278/128a3554-82c4-4184-ba2a-1f9602d2a8d6)

This is a functionality regression.

**Comment by michaldrabik** [comment]
If you are signed to Trakt the app will verify this on the spot while you create lists. You can "hack" it by logging out of Trakt and then you can create as many lists as you want. However if you want to sync with Trakt not everyone of them is gonna by synced due to Trakt limits.

**Comment by kranix0** [comment]
Thanks. I logged out of Trakt and my extra lists became visible. I added another and then logged into Trakt. I can still see the extra lists.

I understand that the additional lists won't get synced to Trakt. Can they be backed up anywhere so that I will still have them if I lose my phone or uninstall the app?

**Comment by skst** [comment]
You can export your Showly data and save that file.

**Comment by michaldrabik** [comment]
Yes, If you sync with Trakt the limits will be respected. But you can know backup all your data independently.
